### PR TITLE
nit: daemonize.c: fix Stevens' in comment

### PR DESCRIPTION
**Pull request by jsarenik**

### CODE REVIEW

File: ccan/daemonize/daemonize.c [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 
-/* This code is based on Stevens Advanced Programming in the UNIX
+/* This code is based on Steven's Advanced Programming in the UNIX
  * Environment. */
```
Comment (dgibson) [inline ccan/daemonize/daemonize.c:10]: The author's name is "Richard Stevens", not "Richard Steven", so it should be "Stevens'", not "Steven's".

**Review by dgibson — CHANGES_REQUESTED** [review]

File: ccan/daemonize/daemonize.c (hunk 1 above)
Comment (jsarenik) [reply ccan/daemonize/daemonize.c:10]: Thanks. Fixed.

**Comment by jsarenik** [comment]
This patch happened while i was reading https://github.com/ElementsProject/lightning/pull/4163

File: ccan/daemonize/daemonize.c (hunk 1 above)
Comment (jsarenik) [reply ccan/daemonize/daemonize.c:10]: Fixed in 53688a0b

**Review by dgibson — APPROVED** [review]

**Pull request closed without merging**